src/workbench/Workbench.tsx

```diff
@@ -99,15 +113,21 @@ const Workbench = () => {
                 mode={serialSizedMode}
               >
                 <SplitViewRemainder>
-                  <Box height="100%" as="section">
-                    {selection && fileVersion !== undefined && (
-                      <EditorArea
-                        key={selection.file + "/" + fileVersion}
-                        selection={selection}
-                        onSelectedFileChanged={setSelectedFile}
-                      />
-                    )}
-                  </Box>
+                  {flags.simulator ? (
+                    <SplitView
```

Comment:
This split view behaves poorly when nested like this (perhaps due to minimums).

The layout as a whole isn't consistent with the latest designs.

However, so long as it has no effect on the non-flagged layout I think it's worth merging so we can avoid maintaining a separate branch as we iterated on the simulator.

Reply:
Ah, I think it's the equivalent of this bug:

https://github.com/tomkp/react-split-pane/issues/30

So we need to add an overlay element over the iframe (or for simplicity, each side) during the drag.

Reply:
Now I understand it I feel less bad about merging it. I'm going to leave this one to fix afterwards.